designs/tripleo/osc-tripleo.md

```diff
@@ -0,0 +1,137 @@
+# OSC TripleO
```

Comment:
Can't we call the document as OSC Integration with TripleO? we could have different chapters like THT, Puppet (if needed), services etc. I see RH as documentation on partner integration.